acquire-driver-common/src/simcams/simulated.camera.c

```diff
@@ -491,8 +540,10 @@ simcam_close_camera(struct Camera* camera_)
       containerof(camera_, struct SimulatedCamera, camera);
     EXPECT(camera_, "Invalid NULL parameter");
     simcam_stop(&camera->camera);
-    if (camera->im.data)
-        free(camera->im.data);
+    if (camera->im.frame_data)
+        free(camera->im.frame_data);
+    if (camera->im.render_data)
+        free(camera->im.render_data);
```

Comment:
Probably worth searching around for other instances of these. 

```suggestion    
    free(camera->im.frame_data);    
    free(camera->im.render_data);
```